tests_recording/testbase.py

```diff
@@ -30,7 +31,7 @@ def get_test_config():
         return conf
 
     def get_datafile_filename(self, suffix="yaml"):
-        prefix = PERSISTENT_DATA_PREFIX
+        prefix = str(PERSISTENT_DATA_PREFIX)
```

Comment:
I think you can work with it as with a `Path` and this method can return `Path` as well,
see https://github.com/packit-service/packit-service/blob/master/tests_requre/openshift_integration/base.py#L16